asset-transfer-basic/application-gateway-typescript/src/app.ts

```diff
@@ -4,17 +4,38 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+// Pre-requisites:
+// - fabric-sample two organization test-network setup with two peers and ordering service and and 2 certificate authorities
+//         ===> from directory /fabric-samples/test-network
+//         ./network.sh up createChannel
+// - Use any of the asset-transfer-basic chaincodes deployed on the channel "mychannel"
+//   with the chaincode name of "basic". The following deploy command will package,
+//   install, approve, and commit the javascript chaincode, all the actions it takes
+//   to deploy a chaincode to a channel.
+//         ===> from directory /fabric-samples/test-network
+//         ./network.sh deployCC -ccn basic -ccp ../asset-transfer-basic/chaincode-typescript/ -ccl typescript
+// - Be sure that Node.js is installed
+//         ===> from directory /fabric-samples/asset-transfer-basic/application-typescript
```

Comment:
```suggestion
//         ===> from directory /fabric-samples/asset-transfer-basic/application-gateway-typescript
```

Reply:
I didn't change this text, just moved its location in the file. Changes and the build fix will need to wait for another day